mappings/net/minecraft/world/World.mapping

```diff
@@ -215,6 +215,9 @@ CLASS aiw net/minecraft/world/World
 		ARG 0 packet
 	METHOD a getEntitiesIn (Ljava/lang/Class;Lbcs;)Ljava/util/List;
 	METHOD a getEntitiesIn (Ljava/lang/Class;Lbcs;Lcom/google/common/base/Predicate;)Ljava/util/List;
+	METHOD a filterEntities (Ljava/lang/Class;Lcom/google/common/base/Predicate;)Ljava/util/List;
```

Comment:
That's also intended to be functional-style.

Don't apply names just because it matches some class used in a massively different way.